packages/server/.env.example

```
@@ -7,6 +7,7 @@ DATABASE_PASSWORD=upswyng123
 DATABASE_SESSION_SECRET=
 DATABASE_URL=mongodb://localhost:27017/upswyng-dev
 DATABASE_USER=upswyng-dev-user
+GOOGLE_MAPS_API_KEY=<example google maps api key>
```

```suggestion
GOOGLE_PLACES_API_KEY=<example google places api key>
```
May as well be a bit more specific here.